Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -396,6 +400,54 @@ RDKit::SparseIntVect<std::uint32_t> *MorganFingerprintHelper(
   }
   return res;
 }
+
+std::pair<double,double> BCUT2D_list(const RDKit::ROMol &m, python::list atomprops)
```

Comment:
I believe this block also needs to be in an `#ifdef RDK_HAS_EIGEN3` or it won't compile